network.nix

```diff
@@ -45,8 +45,9 @@ in {
       bitcoin-node = import ./configuration.nix;
     in {
       deployment.keys = {
-        inherit bitcoin-rpcpassword lightning-charge-api-token;
+        inherit bitcoin-rpcpassword;
       }
+      // (if (config.services.lightning-charge.enable) then { inherit lightning-charge-api-token; } else { })
```

Comment:
How did this prevent disabling clightning?

Reply:
The problem was that before it would always deploy the `lightning-charge-api-token` and set the `clightning` user to the owner even if `clightning` wasn't enabled. However, if `clightning` isn't enabled, there's no `clightning` user.